plugins/foreman_ansible/2.x/index.md

```diff
@@ -0,0 +1,210 @@
+
+After your host is provisioned and calls home (by using `foreman_url('built')` in a template), Foreman will do an initial Ansible run to configure the host according to the applicable roles for it. Since Foreman cannot know how long it will take until the host comes back online after the OS is installed and rebooted, Foreman will make Ansible timeout for 5 minutes. This is configurable under Administer > Settings, 'Post-provision timeout' or on a per-host basis by overriding the setting with a parameter.
+
+Inherited roles are added automatically upon host group change and the select interface disables direct removal of such items.
+![select roles for host](static/images/plugins/foreman_ansible/select_roles_for_host.png)
+
+Host group edit form contains very similar tab where you can pre-configure selection of Ansible roles for hosts.
+![select roles for hostgroup](static/images/plugins/foreman_ansible/select_roles_for_hostgroup.png)
+
+## 4.3 Running Ansible Roles
+
+To run the playbook with the roles selected, click on the 'play Ansible roles' button and Foreman will run a playbook on the background.  Logs for this playbook can be found under the regular Foreman log, usually `/var/log/foreman/production.log`
+
+![role play](static/images/plugins/foreman_ansible/role_play.png)
+
+You can also select multiple hosts on the hosts index page, and run the playbook for their Ansible roles by clicking on this button:
+
+![multiple role play](static/images/plugins/foreman_ansible/multiple_role_play.png)
+
+Alternatively, you can run the playbook for all Ansible roles applicable to hosts in a host group by clicking on the Play roles button on the Host groups page.
+
+![hostgroup role play](static/images/plugins/foreman_ansible/hostgroups_role_play.png)
+
+## 4.4 Running Ansible Playbooks (Job Templates)
+
+To run a playbook:
+  - On the Host page, click on Scheduled Remote Job, choose a Job Template, and submit it
+  - On the Host list, select multiple hosts, then click on "Select Action" -> "Scheduled Remote Job"
+  - On the Job Templates list, look for a Job Template and click on "Run", then submit it
+
+For example, here is how to run 'yum install -y vim' once, on one host.
+
+![run ansible job](static/images/plugins/foreman_ansible/run_ansible_job.gif)
+
+And here is how to schedule a playbook run in the future on multiple hosts:
+
+![schedule ansible job](static/images/plugins/foreman_ansible/schedule_ansible_job.gif)
+
+For more information about scheduled jobs, check out the [Foreman Remote Execution manual](/plugins/foreman_remote_execution)
+
+## 4.5 Overriding Ansible Variables
+
+To override an Ansible variable, set the variable you want to override as a Host Parameter, Host Group Parameter, or Global Parameter. The 'name' of the parameter would be the name of the Ansible variable you want to override, and the 'value' of the parameter the value of the variable.
+
+<!--
```

Comment:
@ares It seems that this text is still rendered even though it's commented

Reply:
it's rendered but I don't see it in browser, so it's fine